library/pkparse.c

```diff
@@ -487,6 +486,62 @@ static int pk_use_ecparams(const mbedtls_asn1_buf *params, mbedtls_ecp_group *gr
     return 0;
 }
 
+#if defined(MBEDTLS_ECP_DP_CURVE25519_ENABLED) || defined(MBEDTLS_ECP_DP_CURVE448_ENABLED)
```

Comment:
I just took a quick look at this PR. I have question for @mpg concerning this guards: following the work done in #7103 and since parsing is available both for the software implementation as well as the PSA one, shouldn't these guards be replaced with `PSA_WANT_`?

Reply:
Good point. However I'm not convinced this is a problem right now, as opposed to in the near future when we progress the [2c EPIC](https://github.com/orgs/Mbed-TLS/projects/1?card_filter_query=is%3Aopen#column-19391691) - which would make it not @jethrogb's problem but ours/yours ;)

I'd recommend we wait and see what the CI has to say about it once the PR has been rebased on development - if it passes without a change here, this is a problem for later, otherwise it has to be fixed before merging this PR obviously.